### PR TITLE
feat(sample): sample of geometry

### PR DESCRIPTION
**Pull request by hellmor**
Add sample about geometries

### CODE REVIEW

File: samples/geometry/Sample_VertexAnimation.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        let renderer = floor.addComponent(MeshRenderer);
+        renderer.geometry = this.floorGeometry;
+        renderer.material = new LitMaterial();
+        this.scene.addChild(floor);
+    }
+
+    private update() {
+        // update its vertex data over time
+        if (this.floorGeometry) {
+            let attribute = this.floorGeometry.getAttribute(VertexAttributeName.position);
```
Comment (sirbignian) [inline samples/geometry/Sample_VertexAnimation.ts:57]: It's best to do water wave calculations here

**Review by Codeboy-cn — APPROVED** [review]

File: samples/geometry/Sample_VertexAnimation.ts (hunk 1 above)
Comment (hellmor) [reply samples/geometry/Sample_VertexAnimation.ts:57]: Please see line 60.
It's already look like a wave.

Comment (hellmor) [reply samples/geometry/Sample_VertexAnimation.ts:57]: image

Comment (hellmor) [reply samples/geometry/Sample_VertexAnimation.ts:57]: image

after computeNormals

**Review by sirbignian — APPROVED** [review]

**Pull request merged**